lib/version_utils.pm

```diff
@@ -169,7 +166,7 @@ sub is_caasp {
     elsif ($filter =~ /\d\.\d\+?$/) {
         # If we use '+' it means "this or newer", which includes tumbleweed
         return ($filter =~ /\+$/) if check_var('VERSION', 'Tumbleweed');
-        return check_version($filter, qr/\d\.\d/);
+        return check_version($filter, get_var('VERSION'), qr/\d\.\d/);
```

Comment:
There is one use-case we kind of missed, which was supported by `sle_version_at_least`, that it works with any variable, not just `VERSION`. Could be reused for HDD_VERSION, as migration team still uses sle_version_at_least. WDYT?

Reply:
This is `is_caasp`, we don't have this use-case for now.
I implemented what you are asking for in `is_sle`. You can use second parameter:
`is_sle('12+', get_var('HDD_VERSION'))`.

This should allow to remove `sle_version_at_least`

Reply:
Awesome! Thanks for that!